apps/video/test/marionette/lib/video.js

```diff
@@ -192,6 +195,66 @@ Video.prototype = {
   },
 
   /**
+   * Taps 'pause' button and checks that the counter is not progressing.
+   * returns false if the counter has progressed.
+   */
+  pausePlayback: function() {
+    this.client.helper.waitForElement(Video.Selector.playpauseButton).click();
+    var currentTime = this.getElapsedTimeSeconds();
+    
+    // wait for 2 seconds
+    this.client.executeAsyncScript(function () 
+      {setTimeout(function(){
+        marionetteScriptFinished();
+      }, 2000);
```

Comment:
ah, so you're saying that by moving that code, it ensures that by the time the icon is changed the playback is actually paused.  hmm, may be a better idea. will try.